.travis.yml

```diff
@@ -1,3 +1,4 @@
+sudo: false
```

Comment:
For Travis [container env](https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments)